src/registrar/templates/home.html

```diff
@@ -1,59 +1,106 @@
 {% extends 'base.html' %}
 
+{% load static %}
+
 {% block title %} Hello {% endblock %}
-{% block hero %}
-<section class="usa-hero">
-  <div class="usa-grid">
-    <div class="usa-hero-callout usa-section-dark">
-      <h1>
-        <span class="usa-hero-callout-alt">Welcome to the .gov registrar</span>
-      </h1>
-    </div>
-</section>
-{% endblock %}
 
 {% block content %}
 <main id="main-content" class="grid-container">
-<p>This is the .gov registrar.</p>
-
 {% if user.is_authenticated %}
+{# complete logged in page together here #}
+
+<div class="tablet:grid-offset-1 desktop:grid-offset-2">
+  <h1>Manage your domains</h2>
+
+  <p><a href="{% url 'application:' %}" class="usa-button">Start a new domain request</a></p>
+
+  <section class="dashboard tablet:grid-col-11 desktop:grid-col-10">
+    <h2>Registered domains</h2>
+    {% if domains %}
+      {% for domain in domains %}
+      {{ domain }}
+      {% endfor %}
+    {% else %}
+      <p>You don't have any registered domains yet</p>
+    {% endif %}
+  </section>
 
-{% if domain_applications %}
-<h2>Your domain applications</h2>
-<table class="usa-table usa-table--borderless">
-  <caption class="sr-only">Your domain applications</caption>
-  <thead>
-    <tr>
-      <th>Name</th>
-      <th>Status</th>
-    </tr>
-  </thead>
-  <tbody>
-    {% for application in domain_applications %}
-      <tr>
-        <th>
-          <a href="{% url 'edit-application' application.pk %}">
+  <section class="dashboard tablet:grid-col-11 desktop:grid-col-10">
+    <h2>Active domain requests</h2>
+    {% if domain_applications %}
+    <table class="usa-table usa-table--borderless usa-table--stacked">
+      <caption class="sr-only">Your domain applications</caption>
+      <thead>
+        <tr>
+          <th data-sortable scope="col" role="columnheader">Domain name</th>
+          <th data-sortable scope="col" role="columnheader">Date created</th>
+          <th data-sortable scope="col" role="columnheader">Status</th>
+          <th role="columnheader"><span class="usa-sr-only">Action</span></th>
+        </tr>
+      </thead>
+      <tbody>
+        {% for application in domain_applications %}
+        <tr>
+          <th th scope="row" role="rowheader" data-label="Domain name">
             {{ application.requested_domain.name|default:"New domain request" }}
-          </a>
-        </th>
-        <td>{{ application.status }}</td>
-      </tr>
+          </th>
+          <td data-sort-value="{{ application.created_at|date:"U" }}" data-label="Date created">{{ application.created_at|date }}</td>
+          <td data-label="Status">{{ application.status|title }}</td>
+          <td>
+            <a href="{% url 'edit-application' application.pk %}">
+              {% if application.status == "started" %}
+              <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
```

Comment:
**Suggestion (non-blocking)**: These `svg`s are pretty big without CSS; let's constrain their size. (Examples are elsewhere in the codebase.)